translations/de/info/task_description.html

```diff
@@ -0,0 +1,79 @@
+    und "Strom fliesst nicht" in einem Schaltkreis entsprechen, typischerweise mit 1 respektiv 0 dargestellt.
+    Im Unterschied zur elementaren Algebra, deren Variablen Werte als Zahlen besitzen mit den Grundoperationen Addition und
+    Multiplikation, sind die Grundoperationen der boolschen Algebra die Konjunktion (∧), Disjunktion (∨) und Negation (¬).
+</p>
+
+<p>
+    In dieser Mission kannst Du einige der boolesche Operationen implementieren:<br>
+    - <strong>"Konjunktion"</strong>
+    Ausdruck x&nbsp;∧&nbsp;y, es ist x&nbsp;∧&nbsp;y = 1 nur wenn x = y = 1, andernfalls x&nbsp;∧&nbsp;y = 0.<br>
+    - <strong>"Disjunktion"</strong>
+    Aussdruck x&nbsp;∨&nbsp;y, es ist x&nbsp;∨&nbsp;y = 0 nur wenn x = y = 0, andernfalls x&nbsp;∨&nbsp;y = 1.<br>
+    - <strong>"Implikation"</strong> (materiale Implikation)
+    Ausdruck x&rarr;y kann auch als ¬&nbsp;x&nbsp;∨&nbsp;y geschrieben werden.
+    Wenn x wahr ist, dann muss für x → y = 1 auch y wahr sein. Wenn aber x falsch ist, kann y ignoriert werden, und der Ausdruck ist immer wahr.<br>
+    - <strong>"ausschließende Disjunktion"</strong> (ausschließendes Oder)
+    Ausdruck x&nbsp;⊕&nbsp;y kann auch als (x&nbsp;∨&nbsp;y)∧&nbsp;¬&nbsp;(x&nbsp;∧&nbsp;y) geschrieben werden.
+    Schliesst die Möglichkeit von x und y aus. Als arithmetischer Ausdruck ist es eine Addition und Division von 2 mit Rest, 
+    wobei 1 + 1 = 0 ist.<br>
+    - <strong>"Äquivalenz"</strong>(materiale Äquivalenz, Bikonditonal)
+    Ausdruck x&nbsp;≡&nbsp;y, kann auch als ¬&nbsp;(x&nbsp;⊕&nbsp;y) geschrieben werden.
+    Ist nur dann wahr, wenn x und y den selben Zustand haben.<br>
+
+</p>
+
+<p>
+   So sieht die Wahrheitstabelle für diese Operationen aus:
+</p>
+<div style="text-align: center;">
+<pre>
+ x | y | x∧y | x∨y | x→y | x⊕y | x≡y |
+--------------------------------------
+ 0 | 0 |  0  |  0  |  1  |  0  |  1  |
+ 1 | 0 |  0  |  1  |  0  |  1  |  0  |
+ 0 | 1 |  0  |  1  |  1  |  1  |  0  |
+ 1 | 1 |  1  |  1  |  1  |  0  |  1  |
+--------------------------------------
+</pre>
+</div>
+
+<p>
+    Du erhälst je einen booleschen Wert für <strong>x</strong> und <strong>y</strong> sowie eine der eben beschriebenen Operationen. 
+    Du musst den resultierenden Zustand berechnen und als 1 oder 0 zurückgeben.
+</p>
+<p>
+    <strong>Eingabe: </strong> Drei Argumente. x und y als 0 oder 1. Der Name der Operation als String.
+</p>
+
+<p>
+    <strong>Ausgabe: </strong> Das Ergebnis als 1 oder 0.
+</p>
+
+
+<div class="for_info_only">
+    <p>
+        <strong>Beispiel:</strong>
+    </p>
+    <pre class="brush: python">
+boolean(1, 0, "conjunction") == 0
+boolean(0, 1, "exclusive") == 1
+    </pre>
+</div>
+
+<p class="for_info_only">
+    <strong>Wozu das gut ist: </strong>
+    Du lernst die allen Arbeitsschritten eines Computers zugrundeliegenden booleschen Werte und Operationen kennen,
+    wie in diesem Artikel zu <a href="https://de.wikipedia.org/wiki/Logikgatter>Logikgatter</a> weiterführend beschrieben.
```

Comment:
- **Von:** _wie in diesem Artikel zu `<a href="https://de.wikipedia.org/wiki/Logikgatter>`Logikgatter`</a>` weiterführend beschrieben._  
 **Zu:** _Aus diesem Artikel zu `<a href="https://de.wikipedia.org/wiki/Logikgatter>`Logikgatter**n**`</a>` kannst Du sogar noch mehr lernen._  
 ==> irgendwie schöner... ?!?!